app/models/course/group_user.rb

```diff
@@ -0,0 +1,12 @@
+class Course::GroupUser < ActiveRecord::Base
+  stampable
+  belongs_to :user, inverse_of: :group_users
```

Comment:
...actually, does it matter? Under what circumstances do we need to have a CourseUser object?